tests/roles/ceph_migrate/tasks/ceph_validate.yaml

```diff
@@ -60,8 +61,8 @@
 
     - name: ansible.builtin.fail if Ceph <= Quincy
       ansible.builtin.fail:
-        msg: "Ceph version is <= Reef"
-      when: ceph.monmap.min_mon_release_name != "quincy"
+        msg: "Ceph version is != Reef"
+      when: ceph.monmap.min_mon_release_name != "reef"
```

Comment:
do we need to update docs https://raw.githubusercontent.com/marios/data-plane-adoption/refs/heads/main/docs_user/modules/proc_migrating-ceph-mds.adoc  i saw reference to quincy

Reply:
we already tracked somewhere else with @klgill, and all the references are going to be updated as part of a doc patch (mostly because we explicitly stated Ceph Reed and the upgrade from 6 to 7 is a requirement).
I'll proactively start a patch to update all the references.

Reply:
ack thanks